tracing/plugin/otlp.go

```diff
@@ -20,53 +20,34 @@ import (
 	"context"
 	"fmt"
 	"io"
+	"net/url"
 	"time"
 
 	"github.com/containerd/containerd/log"
 	"github.com/containerd/containerd/plugin"
 	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/exporters/otlp/otlptrace"
 	"go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracegrpc"
+	"go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracehttp"
 	"go.opentelemetry.io/otel/propagation"
 	"go.opentelemetry.io/otel/sdk/resource"
 	sdktrace "go.opentelemetry.io/otel/sdk/trace"
 	semconv "go.opentelemetry.io/otel/semconv/v1.4.0"
-	"google.golang.org/grpc"
 )
 
 const exporterPlugin = "otlp"
 
 func init() {
-	const timeout = 5 * time.Second
-
 	plugin.Register(&plugin.Registration{
 		ID:     exporterPlugin,
 		Type:   plugin.TracingProcessorPlugin,
 		Config: &OTLPConfig{},
 		InitFn: func(ic *plugin.InitContext) (interface{}, error) {
 			cfg := ic.Config.(*OTLPConfig)
 			if cfg.Endpoint == "" {
-				return nil, fmt.Errorf("otlp endpoint not set: %w", plugin.ErrSkipPlugin)
-			}
-
-			opts := []otlptracegrpc.Option{
-				otlptracegrpc.WithEndpoint(cfg.Endpoint),
-				otlptracegrpc.WithDialOption(
-					grpc.WithBlock(),
-					grpc.WithReturnConnectionError(),
-				),
-			}
-			if cfg.Insecure {
-				opts = append(opts, otlptracegrpc.WithInsecure())
+				return nil, fmt.Errorf("OTLP endpoint not set: %w", plugin.ErrSkipPlugin)
```

Comment:
```suggestion
				return nil, fmt.Errorf("OpenTelemetry endpoint not set: %w", plugin.ErrSkipPlugin)
```

for consistency with the `newExporter`

Reply:
done